src/components/organisms/PaneManager/PaneManagerLeftMenu.tsx

```diff
@@ -94,7 +86,7 @@ const PaneManagerLeftMenu: React.FC = () => {
       >
         <MenuButton
           id="file-explorer"
-          isSelected={!leftDrawerVisible && leftMenuSelection === 'file-explorer'}
+          isSelected={Boolean(activeProject) && leftMenuSelection === 'file-explorer'}
```

Comment:
`leftMenuSelection === 'file-explorer'` => `leftMenuSelection === Enum.FileExplorer` something like this
I know that this was not your doing.. I prefer to use enums instead of strings even if they are mapped to types.
Enums offer a lot more context at a glance and it is a lot easier to find usages and improve code readability

Reply:
I will leave them unchanged for now, we can maybe do a tech debt refactoring of using Enums.